lua/octo/util.lua

```diff
@@ -222,31 +222,54 @@ function M.get_comment_at_cursor(bufnr, cursor)
     local start_line = mark[1] + 1
     local end_line = mark[3]["end_row"] + 1
     if start_line <= cursor[1] and end_line >= cursor[1] then
-      return {comment, start_line, end_line}
```

Comment:
And here comes test to check how a suggestion looks like :upside_down_face:

```suggestion
      return { comment, end_line, start_line }
```

Reply:
Okay I can see the plain "source". The fancy approach would be to replace the (three backtics +) `suggestion` with the file ending of the file that got commented. In markdown this should make it get highlighted with the proper syntax if the user has enabled this. Else, as GitHub does it show the actual comparison between both lines with red and green. 
But this is again a nice to have for later addition. Works already how it is.

Reply:
Handled in issue #109

Conversation can be closed (I can't close it...).